docs/substrate-based-crab-chain/crab-tut-create-account.md

```diff
@@ -5,102 +5,36 @@ sidebar_label: Create an Account
-
-- Click [Start generation], after waiting for the account to be generated, click [+] to select your favorite account for backup.
-
-![faucet](../assets/substrate-based-crab-chain/create-account-09.png)
-
-- Click [Save], according to your usage habits, choose to back up the original seed or json file
-
-![faucet](../assets/substrate-based-crab-chain/create-account-10.png)
-
-> Be sure to back up `mnemonics, private keys, json files`, etc. When backing up the json file, please keep the password safe. If the password is lost, the address cannot be restored through the json file, but it can be re-imported through the mnemonic word and private key.
-
-
-
-## 2. Polkadot.js Browser Plugin
-
-**Install the Browser Plugin**
-
-The browser plugin is available for both [Google Chrome](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd?hl=en) and [FireFox](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension).
-
-![polkadot-js](../assets/substrate-based-crab-chain/create-account-11.png)
-
-**New Account**
-
-Click the extension to open the "Account Management" dialog box, click the "Create New Account" button, and then follow the instructions.
-
-![polkadot-js](../assets/substrate-based-crab-chain/create-account-12.png)
-
-![polkadot-js](../assets/substrate-based-crab-chain/create-account-13.png)
-
-![polkadot-js](../assets/substrate-based-crab-chain/create-account-14.png)
-
-> Make sure to keep the mnemonics safe.
-
-**Set Address for Darwinia Mainnet or Crab Network**
-
-Now we will ensure that the addresses are displayed as Darwinia mainnet addresses.  Your address will be different depending on network selection.
-
-Click on "Options" at the top of the plugin window.  Select `Crab Network` or `Substrate` in "Display Address Format for" dropdown box. 
-
-![polkadot-js](../assets/substrate-based-crab-chain/create-account-15.png)
-
-> Crab Network share the same `Network ID` as `Substrate`, if you need to choose a network when generating an account, use `substrate` as the same effect as `crab network`.
-
-
-## 3. Subkey
-
-Subkey is recommended for technically advanced users who are comfortable with command line and compiling Rust code. Subkey allows you to generate keys on any device that can compile the code. Subkey may also be useful for automated account generation, using an air-gapped device other than one running iOS or Android or other specific purposes. It is not recommended for general users.
+The Subkey is recommended for technically advanced users who are comfortable with command line and compiling Rust code. Subkey allows you to generate keys on any device that can compile the code. Subkey may also be useful for automated account generation, using an air-gapped device other than one running iOS or Android or other specific purposes. It is not recommended for general users.
 
 To [install Subkey](https://substrate.dev/docs/en/ecosystem/subkey#more-subkey-to-explore), run:
```

Comment:
![image](https://user-images.githubusercontent.com/11801722/159197947-98e2f001-67b4-4e9e-95ce-075861fe9ff7.png)

It seems this link is broken.